### DISCUSSION TITLE
This map does not have a terrain density size supported by TerraFarm, terrain deformation may not work as intended.

### DISCUSSION THREAD

**Discussion by glouver** [scfmod/FS22_TerraFarm#79] [Q&A] [answered]
"This map does not have a terrain density size supported by TerraFarm, terrain deformation may not work as intended."

how to fix it?

(normal map is ok.)
(4x maps have this problem.)

**Comment by scfmod** [comment, marked as answer]
It's not an error, it's just a warning :)
If it doesn't work on the map you probably need to wait for next major release of TerraFarm

**Reply by JohannesSpg** [reply]
Which is never to be published only given to someone i guess. I know his name. Terrafarm is basically stopped.
Now Jeremy Brown forming a group of specialists on Discord where we can use a scripting person.

**Reply by scfmod** [reply]
If you are trying to create some "drame" you can take your guesses somewhere else please. There is no need for you to spread misinformation.
As with all other modders there are situations in life that occurs which makes dedicating time to a hobby project not possible.
I don't know of Jeremy Brown unfortunately, but if he is interested in a dialog or just a friendly chat (or someone else for that matter) I'm available on Discord username: estyx  :-)

**Reply by JohannesSpg** [reply]
Not trying to create drama in any form. I will try to invite you into Jeremy's group where we can have a dialogue about this if you wish. With Jeremy and Mcgee, the two persons to who i exchanged my ideas. Again not creating drama at all.

**Reply by scfmod** [reply]
Yeah I don't mind at all, feel free to contact me on Discord :-) Sorry for that came out a bit cross, that was not my intention and I do apologise for that.

**Reply by JohannesSpg** [reply]
Apologies accepted sir, i not wanted to push myself on you in the last 1,5 years or so.
I tried to find your name in Discord but it says it could not find it.
So i forwarded your name to Jeremy but i think he is on a business trip or traveling or whatever.
Below is the link to invite you directly to Jeremy's Test server.

https://discord.gg/PJ2qZ2xc
